packages/dashboard/src/pages/routes.tsx

```diff
@@ -23,7 +24,11 @@ export function Pages() {
         <Suspense fallback="loading...">
             <Switch>
                 <Route path={Routes.Welcome} children={<Welcome />} />
-                <Route path={Routes.Personas} children={frame(<Personas />)} exact />
+                <Route
+                    path={Routes.Personas}
+                    children={<PersonaState.Provider>{frame(<Personas />)}</PersonaState.Provider>}
```

Comment:
I think it's better to add inside `<Personas />`

Reply:
I'm afraid this won't work, as `PersonaState.useContainer()` is used in `Personas`. So the Provider wraps the `Personas` component